### PR TITLE
QMS - LTSS-6, LTSS-7, LTSS-8 - Measure Overview Page

### PR DESCRIPTION
**Pull request by ajaitasaini**
### Description

This PR adds the content for the LTSS-6, LTSS- 7, LTSS-8 measure overview page. The content of what's suppose to be on this page is written in the Jira ticket, the Figma is just for a general idea of what the page might look like

Notes from LTSS-3:
"I propose, holding off on the edit button going to a blank page at least until the routing by page id PR is merged in, so there's consistency with how we're routing."

Notes from LTSS-1:
"Children textboxes of the radio button currently does not save any data. Address in a separate PR."
"The quality measure table [Edit] buttons do not go anywhere yet. Address in a separate PR."

### Related ticket(s)

CMDCT-4149, CMDCT-4151, CMDCT-4153

---
### How to test

Sign into HCBS, any state user: https://d1n5pfglnxp0wk.cloudfront.net/
Create a QM report
On the sidebar menu, select [Required Measure Results]. You should see a row for CMIT # 20.
The generation for this table is covered by another ticket.
Click [Edit] and you be taken to the LTSS-6 measure overview page.
Check that the questions matches what is written in the jira ticket: [CMDCT-4141](https://jiraent.cms.gov/browse/CMDCT-4141).
You should see a page that looks like this:
Screenshot 2024-12-11 at 2 08 45 PM

Repeat steps for LTSS-7 (you should see the same questions/page but with CMIT # 968
Repeat steps for LTSS-8 (you should see the same questions/page but with CMIT # 414

### Important updates

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by BearHanded — DISMISSED** [review]

**Review by rocio-desantiago — CHANGES_REQUESTED** [review]
Remove the question: "What Technical Specifications are you using to report this measure?" 
That one only applies to LTSS-1, LTSS-2, LTSS-3, LTSS-4 Measures

File: services/app-api/forms/cmit.ts [hunk 1]
```diff
@@ -58,4 +58,43 @@ export const CMIT_LIST: CMIT[] = [
...
+    cmit: 20,
+    name: "LTSS-6: Admission to a Facility from the Community",
+    uid: "20",
+    measureSteward: "CMS",
+    measureSpecification: [
+      MeasureSpecification.CMS,
+      MeasureSpecification.HEDIS,
+    ],
+    deliverySystem: [DeliverySystem.FFS, DeliverySystem.MLTSS],
+    dataSource: DataSource.Hybrid,
```
Comment (rocio-desantiago) [inline services/app-api/forms/cmit.ts:71]: Found out that LTSS-6 has an Administrative dataSource in this [file](https://docs.google.com/spreadsheets/d/1ztZ1DSYCUkKz9E_np5q-k9CgBoHYKEWUc_Fd2t3RJmc/edit?gid=1582150716#gid=1582150716)

**Review by rocio-desantiago — CHANGES_REQUESTED** [review]
From this [excel sheet](https://docs.google.com/spreadsheets/d/1ztZ1DSYCUkKz9E_np5q-k9CgBoHYKEWUc_Fd2t3RJmc/edit?gid=1582150716#gid=1582150716) - that should have been on the ticket lol
measure spec is only CMS for all 3, delivery system still has both MLTSS and FSS and Administrative instead of Hybrid Data source

**Review by rocio-desantiago — APPROVED** [review]
Looks good 👍

**Review by angelaco11 — APPROVED** [review]
Looks good to me!

**Pull request merged**